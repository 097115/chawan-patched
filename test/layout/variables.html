<style>
a { --purple: purple; color: var(--purple) }
div { color: white; --pink: yellow !important; background-color: var(--purple) }
</style>
<a>
<div>purple</div>
</a>
