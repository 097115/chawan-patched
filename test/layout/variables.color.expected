[38;2;255;255;255m[48;2;128;0;128mpurple[39m                                                                          [49m

